checkpoint/client/rest/query.go

```diff
@@ -406,15 +406,44 @@ func latestCheckpointHandlerFunc(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
+		var checkpointUnmarshal hmTypes.Checkpoint
```

Comment:
check if you can use checkpointWithID instead of creating a new var checkpointUnmarshal

var checkpointWithID hmTypes.CheckpointWithID
json.Unmarshal(res, &checkpointWithID)

and then set 
checkpointWithID.ID = ackCount

Reply:
Yes, I checked. We can use it and will update this change.